gallery/src/index.css

```
@@ -0,0 +1,111 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    color: white;
+}
+
+.restricted-media {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.connection-banner,
+.main-window,
+.info-box {
+    border: 1px solid #308274;
+    border-radius: 10px;
+    background: black;
+}
+
+.connection-banner {
+    padding: 0 10px;
+    max-width: 50%;
+    width: 50%;
```

```suggestion
    width: 80%;
    max-width: 1200px;
```

And then apply that to `.main-window` as well.